### DISCUSSION TITLE
Paid version

### DISCUSSION THREAD

**Discussion by PrathameshSolanke** [lllyasviel/Fooocus#2994] [Q&A] [answered]
I am thinking about getting paid version will it have every updates as soon as they are released like the recent mid journey update and 2.4.0 rc

**Comment by ILikeToasters** [comment, marked as answer]
What do you mean by paid version? There are some services like Colab and Rundiffusion and some others that you can use Fooocus on but if you are talking about the sites that claim to be Fooocus and charge they are scams. And the Fooocus project here on Github has no association with any of those.  There is no "paid version".

**Reply by PrathameshSolanke** [reply]
Thanks for letting me know ☺️.
